### ISSUE TITLE
Error: Inline JavaScript is not enabled. Is it set in your options? (Minifier for LESS)

### ISSUE THREAD

**Issue by saesrpg** [NodeBB/NodeBB#10040]
With the below change, I can't get our custom theme to build unless I manually change this from false back to true

https://github.com/NodeBB/NodeBB/commit/28efcb59cbcd301424f58dffeb6e86a0f213095a#diff-c10484641c685f21743db91ce1b091c60d9f393db554625de477b730bd78e03eR231

Would it be possible to move this option to the theme itself?

**Comment by julianlam** [comment]
No unfortunately not, as it was disabled on purpose due to a potential security issue.

Can you isolate what exactly is the inline JavaScript you are running?

**Comment by julianlam** [comment]
In our case it was an odd bit of LESS that wasn't actually inline js, and also was longer required due to better browser support.

**Comment by julianlam** [comment]
> Would it be possible to move this option to the theme itself?

Ah I misread. We _could_ potentially move this to the theme as a configurable option, but usually it is not a flag that is required to be set true.

**Comment by saesrpg** [comment]
I see, I'll have to see if I can go through the theme files and isolate the inline code.

Any suggestions on what specific syntax to look for?

**Comment by julianlam** [comment]
Sure!

[This is the change we made so that LESS no longer required `inlineJavascript` to be enabled](https://github.com/NodeBB/nodebb-theme-persona/commit/e14d38b827591c9681a5fcd43ca4b206297eaff5)

Bootstrap also has an opacity mixin, but theirs is slightly different and doesn't seem to trigger this issue. Nevertheless, opacity is basically supported cross-browser and so we removed it altogether from Persona.
